nucleus/parent/pom.xml

```diff
@@ -71,14 +71,14 @@
     </issueManagement>
     <properties>
         <!-- Jakarta Expression Language -->
-        <jakarta.el-api.version>5.0.1</jakarta.el-api.version>
+        <jakartaElApiVersion>5.0.1</jakartaElApiVersion>
```

Comment:
Please don't do that.

Reply:
- https://github.com/eclipse-ee4j/glassfish/issues/25642 explicitly shows NEW PROPERTY introduced, to be used in docs, and old property kept - then no change in any other pom would be necessary.
- I'd use `jakarta-el-api-version` in guide instead.